library/common/stats/engine_utility.cc

```diff
@@ -0,0 +1,25 @@
+// From upstream Envoy
+#include "common/stats/utility.h"
+
+#include "library/common/http/header_utility.h"
+#include "library/common/types/c_types.h"
+
+namespace Envoy {
+namespace Stats {
+namespace Engine {
+namespace Utility {
+
+Stats::StatNameTagVector transform(envoy_stats_tags tags, Stats::StatNameSetPtr& stat_name_set) {
```

Comment:
nit: should we have a more specific name? we generally use a more specific name for utilities like this e.g., `copyToString`

Reply:
(Also, please say "copy" only if we're creating a copy.)

Reply:
my assumption was that the parameter(s), the return type and the function name put together makes the intention clear, but let me update it to `transformToStatNameTagVector` - or lmk what would be the name you prefer :)